docs/decisions/0013-custom-groups-for-enterprise-learners.rst

```diff
@@ -0,0 +1,152 @@
+0013 Creating Custom Groups for Enterprise Customers
+##############################
+
+Status
+******
+
+**Pending**
+
+Terminology
+*******
+*Enterprise Group* - Collection of enterprise customer users connected to a policy/subsidy
```

Comment:
Groups are a feature of an enterprise more than a feature of learner-credit. Groups will be used for lots of things like access control, analytics segmentation, bulk actions throughout the enterprise admin experience, etc. I'd caution us against pigeonholing groups into one thing like "...collection of enterprise customer users connected to a policy/subsidy".